apps/admin/backend/src/tabulation/write_ins.test.ts

```diff
@@ -405,7 +405,7 @@ test('modifyElectionResultsWithWriteInSummary', () => {
         tally: 11,
       },
     },
-    undervotes: 15,
+    undervotes: 6,
```

Comment:
Because the write-in tabulation doesn't add undervotes to the tally anymore, the number goes down here.